avm/res/web/static-site/main.bicep

```diff
@@ -274,10 +274,10 @@ resource staticSite_roleAssignments 'Microsoft.Authorization/roleAssignments@202
   }
 ]
 
-module staticSite_privateDnsZone 'br/public:avm/res/network/private-dns-zone:0.7.0' = if (!empty(privateEndpoints) && createPrivateDnsZone == 'Enabled') {
+module staticSite_privateDnsZone 'br/public:avm/res/network/private-dns-zone:0.8.0' = if (!empty(privateEndpoints) && createPrivateDnsZone == 'Enabled') {
   name: '${uniqueString(deployment().name, location)}-staticSite-PrivateDnsZone'
   params: {
-    name: 'privatelink.${staticSite.properties.defaultHostname}.azurestaticapps.net'
+    name: 'privatelink.${split(staticSite.properties.defaultHostname, '.')[1]}.azurestaticapps.net'
```

Comment:
Hey @ChrisSidebotham assuming the `defaultHostName` has a value of like `"happy-sea-15afae3e.azurestaticwebsites.net"`, would you not need to refer to the `[0]` item of the split array 
`[`
`  happy-sea-15afae3e,`
`  azurestaticwebsites,`
`  net`
`]`

?